addons/project/static/tests/tours/project_burndown_chart_tour.js

```diff
@@ -73,4 +73,5 @@ registry.category("web_tour.tours").add('burndown_chart_tour', {
 }, {
     content: 'The comparison menu is not rendered',
     trigger: ':not(:has(.o_comparison_menu))',
+    isCheck: true,
```

Comment:
This step is actually useless :thinking: `:not(:has(.o_comparison_menu))` will always be true, or is there something I am missing? Anyway, it's not related to your PR, let's ignore that